colors/test.ts

```diff
@@ -1,25 +1,121 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 import { test } from "../testing/mod.ts";
 import { assertEquals } from "../testing/asserts.ts";
-import { red, bgBlue, setEnabled, getEnabled } from "./mod.ts";
+import * as _ from "./mod.ts";
```

Comment:
Just a nitpick, could you do `* as c`? `_` always reminds of lodash of unused value

Reply:
Yes will do later today, I aswell wasnt sure about the naming